python-mini-projects/projects/Fetch_open_ports/fetch_open_port.py

```diff
@@ -0,0 +1,17 @@
+from socket import *
+import time
+startTime = time.time()
+
+if __name__ == '__main__':
+   target = input('Enter the host to be scanned: ')
+   t_IP = gethostbyname(target)
+   print ('Starting scan on host: ', t_IP)
+   
+   for i in range(50, 500):
+      s = socket(AF_INET, SOCK_STREAM)
+      
+      conn = s.connect_ex((t_IP, i))
+      if(conn == 0) :
+         print ('Port %d: OPEN' % (i,))
+      s.close()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To close a socket and immediately release its associated resources, call `socket.shutdown()` before you call `socket.close()`.

[Learn more](https://docs.python.org/3/library/socket.html#socket.close)